config/configauth/README.md

```diff
@@ -32,6 +32,6 @@ receivers:
 
 ## Creating an authenticator
 
-New authenticators can be added by creating a new extension that also implements the `configauth.Authenticator` extension. Generic authenticators that may be used by a good number of users might be accepted as part of the core distribution, or as part of the contrib distribution. If you have interest in contributing one authenticator, open an issue with your proposal.
-
+New authenticators can be added by creating a new extension that also implements the `configauth.ServerAuthenticator` or `config.ClientAuthenticator` for server specific authentication or client specific authentication respectively.
```

Comment:
Since ClientAuthenticator is not used anywhere yet I suggest that we don't advertise it in the README yet. Let's add this sentence once ClientAuthenticator is enabled.